lib/bson/bson.js

```diff
@@ -128,6 +128,7 @@ BSON.prototype.serializeWithBufferAndIndex = function(object, finalBuffer, optio
  * @param {Object} [options.promoteValues=false] when deserializing will promote BSON values to their Node.js closest equivalent types.
  * @param {Object} [options.fieldsAsRaw=null] allow to specify if there what fields we wish to return as unserialized raw buffer.
  * @param {Object} [options.bsonRegExp=false] return BSON regular expressions as BSONRegExp instances.
+ * @param {boolean} [options.allowObjectSmallerThanBufferSize] allows the buffer to be larger than the parsed BSON object
```

Comment:
woof, what a property name.. either way let's document that it has a default here as well